src/cgame/cg_drawtools.cpp

```diff
@@ -396,16 +394,12 @@ float CG_FadeAlpha( int startMsec, int totalMsec )
 	// fade out
 	if ( totalMsec - t < FADE_TIME )
 	{
-		fade = ( totalMsec - t ) * 1.0 / FADE_TIME;
+		fade = (float) ( totalMsec - t ) / FADE_TIME;
```

Comment:
nit: Use C++ style casts for new code.

Reply:
I don't really get the point of these fancy casts, and find them unreadable, but :man_shrugging:

Fixed. I've converted one or two extra for good mesure